Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -793,6 +798,23 @@ public extension Purchases {
         return try await logOutAsync()
     }
 
+    @objc func syncAttributesAndOfferingsIfNeeded(completion: @escaping (Offerings?, PublicError?) -> Void) {
```

Comment:
Hmm I've been thinking whether it would be worth moving the logic inside this method to the `PurchasesOrchestrator` (mostly for organization purposes)... However, seems that the `getOfferings` method already was not moved there, so I'm ok with this for now.